packages/marketo-form/partials/fields/select-field/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useMemo } from 'react'
+import { useFormContext } from 'react-hook-form'
+import SelectInput from '@hashicorp/react-select-input'
+import Wrapper from '../../wrapper'
+import type { MarketoFormSelectField, SelectValue } from '../../../types'
+
+function applyVisibilityRule(
```

Comment:
Just a gut check, the visibility rules only apply to select fields right now? Is this feature available for other types of fields?

Reply:
Correct, I discovered recently that we don't use visibility rules in any of our forms, so I'll likely be removing this code in a future iteration. Marketo supports applying visibility rules to any field, but given that we aren't using it currently it doesn't make sense to keep this in.